binding/ios/Porcupine.swift

```diff
@@ -24,6 +24,34 @@ public enum PorcupineError: Error {
     case PorcupineError(_ message:String)
 }
 
+extension PorcupineError : LocalizedError {
+    public var errorDescription: String? {
+        switch self {
+        case .PorcupineMemoryError(let message),
+                .PorcupineIOError(let message),
+                .PorcupineInvalidArgumentError(let message),
+                .PorcupineStopIterationError(let message),
+                .PorcupineKeyError(let message),
+                .PorcupineInvalidStateError(let message),
+                .PorcupineRuntimeError(let message),
+                .PorcupineActivationError(let message),
+                .PorcupineActivationLimitError(let message),
+                .PorcupineActivationThrottledError(let message),
+                .PorcupineActivationRefusedError(let message),
+                .PorcupineError(let message):
+            return NSLocalizedString(message, comment: "")
+        }
+    }
+    
+    public var code : String {
```

Comment:
let's make what we're doing here a bit clearer from an API point of view. Since this is only a get property and it's only for us, we should probably just have a utility function called `getPvException` or something that returns the Exception equivalent